pixi.toml

```diff
@@ -77,9 +77,7 @@ tests = { depends_on = ["lint", "test-ribasim-python", "test-ribasim-core"] }
 generate-schema = { cmd = "julia --project=docs docs/gen_schema.jl", depends_on = [
     "instantiate-julia",
 ] }
-generate-python = """\
-    datamodel-codegen --use-title-as-name --use-double-quotes --disable-timestamp --use-default --strict-nullable --input-file-type=jsonschema --input docs/schema/root.schema.json --output python/ribasim/ribasim/models.py && \
-    datamodel-codegen --use-title-as-name --use-double-quotes --disable-timestamp --use-default --strict-nullable --input-file-type=jsonschema --input docs/schema/Config.schema.json --output python/ribasim/ribasim/config.py"""
+generate-python = "" # "datamodel-codegen --use-union-operator --use-title-as-name --use-double-quotes --disable-timestamp --use-default --strict-nullable --input-file-type=jsonschema --input docs/schema/root.schema.json --output python/ribasim/ribasim/models.py"
```

Comment:
Is that # a typo?
If not, can you please explain it?

Reply:
So the generation of the config.py is removed (I substantially modified these classes). I disabled the generation of the models.py, because using it fails with an odd error message (`E   TypeError: unsupported operand type(s) for |: 'NoneType' and 'NoneType'`) when running ribasim-python. This comes from the use of `--use-union-operator`, resulting in `Something | None = None`, which doesn't seem to work. `Optional[Something] = None` does seem to work, so I'm falling back on that one.